src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/_validators.py

```diff
@@ -34,3 +34,12 @@ def validate_header(string):
         comps = string.split('=', 1)
         result = {comps[0]: comps[1]} if len(comps) > 1 else {string: ''}
     return result
+
+
+def validate_build_task_name(namespace):
+    """Validate the name of the build task name. """
+    import re
+    build_task_name = namespace.build_task_name
+    p = re.compile('^[a-zA-Z0-9]*$')
+    if p.match(build_task_name) == None:
+        raise CLIError("Build task name is not valid.")
```

Comment:
Can you mention why it's not valid and what the acceptable names are? You can use the same message from the sdk.